### PR TITLE
perf: reduce branches.

### PR DESCRIPTION
**Pull request by Et7f3**

### CODE REVIEW

File: src/Cn.re [hunk 1]
```diff
@@ -1,11 +1,13 @@
 let none = "";
 
 let append = (x1, x2) =>
-  switch (x1, x2) {
-  | ("", "") => none
-  | (x1, "") => x1
-  | ("", x2) => x2
-  | (x1, x2) => x1 ++ " " ++ x2
+  switch (x1) {
+  | "" => x2
```
Comment (Et7f3) [inline src/Cn.re:5]: `if x2 == "" then "" else x2` is `x2`

File: src/Cn.re [hunk 2]
```diff
@@ -16,7 +18,7 @@ let fromList = {
   let rec aux = acc =>
     fun
     | [] => acc
-    | [cn, ...rest] when cn == "" => aux(acc, rest)
```
Comment (Et7f3) [inline src/Cn.re:19]: it remove one line in the output: so node seem happier.

Comment (Et7f3) [reply src/Cn.re:19]:
```diff
function fromList(param) {
  var _acc = "";
  var _param = param;
  while(true) {
    var param$1 = _param;
    var acc = _acc;
    if (!param$1) {
      return acc;
    }
-    var rest = param$1[1];
    var cn = param$1[0];
    if (cn === "") {
-      _param = rest;
+      _param = param$1[1];
      continue ;
    }
-    _param = rest;
+    _param = param$1[1];
    _acc = acc + (" " + cn);
    continue ;
  };
}
```

I didn't know declaring variable take time.

**Comment by alexfedoseev** [comment]
Perfect, thanks!

**Pull request merged**